src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int OPERATION_POSITION = 0;
+    static final int AMOUNT_POSITION = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyCount = 0;
+        int buyCount = 0;
+        File fromFile = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file", e);
+        }
+        String[] lines = builder.toString().split(System.lineSeparator());
+        for (String line : lines) {
+            String[] lineSplit = line.split(",");
+            if (lineSplit[OPERATION_POSITION].equals("supply")) {
+                supplyCount += Integer.parseInt(lineSplit[AMOUNT_POSITION]);
+            }
+            if (lineSplit[OPERATION_POSITION].equals("buy")) {
+                buyCount += Integer.parseInt(lineSplit[AMOUNT_POSITION]);
```

Comment:
Consider adding error handling for `Integer.parseInt` to manage potential `NumberFormatException` if the input data is not a valid integer. This will make your code more robust against malformed input data.